plugins/UsersManager/lang/en.json

```diff
@@ -8,7 +8,8 @@
         "AnonymousUserHasViewAccess2": "Your analytics reports and your visitors information are publicly viewable.",
         "ApplyToAllWebsites": "Apply to all websites",
         "ChangeAllConfirm": "Are you sure you want to give '%s' access to all the websites?",
-        "ChangePasswordConfirm": "Changing the password will also change the user's token_auth. Do you really want to continue?",
+        "ChangePasswordConfirm": "Changing the password will also change the user's old API authentication token which may be used to access the Piwik APIs. This token will be re-generated only once. This is a security measure to keep your Piwik data safe. Do you really want to continue?",
+        "ChangePasswordConfirmSelf": "Changing the password will also change your old API authentication token which may be used to access the Piwik APIs. This token will be re-generated only once. This is a security measure to keep your Piwik data safe. Proceeding may require you to log in again. Do you really want to continue?",
```

Comment:
@mneudert it would be great to refactor translation strings so that each sub-string ideally only appears once. This has many advantages especially around making text consistent in different languages and keeping translators working more efficiently :-) For example here, the string can be `Proceeding may require you to log in again.` and reuse `ChangePasswordConfirm`